content/flux/v0.x/stdlib/experimental/bigtable/_index.md

```diff
@@ -7,6 +7,7 @@ description: >
 aliases:
   - /influxdb/v2.0/reference/flux/stdlib/experimental/bigtable/
   - /influxdb/cloud/reference/flux/stdlib/experimental/bigtable/
+  - /flux/v0.65/stdlib/experimental/bigtable/
```

Comment:
@rydermeehan, let's take a look at this one and see if it makes sense to update version to "latest"